tests/Unit/inc/Engine/Admin/Beacon/Beacon/insertScript.php

```diff
@@ -68,9 +64,20 @@ public function testShouldReturnBeaconScript( $locale, $expected ) {
 		$this->options->shouldReceive( 'get_options' )
 			->andReturn( [] );
 
-		$this->assertSame(
-			$this->format_the_html( $expected ),
-			$this->getActualHtml()
-		);
+		$this->setUpGenerate( 'beacon', $expected['data'] );
+
+		$this->beacon->shouldReceive( 'insert_script' )
+			->once();
```

Comment:
This mock is not needed. Why? We're directly invoking in the next lines of code. We know it's going to run once because we are invoking it.